notebooks/tutorials/data-engineer/01-setting-up-dev-mode.ipynb

```diff
@@ -32,13 +148,117 @@
     "## Using Jupyter Environment"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "482b4614",
+   "metadata": {},
+   "source": [
+    "Pysyft has a tox command to set up a local jupyter notebook environment, which is useful for development."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "18ad2a7d",
+   "metadata": {},
+   "source": [
+    "```\n",
+    "tox -e syft.jupyter\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f6fbbe9b",
+   "metadata": {},
+   "source": [
+    "PySyft makes extensive use of jupyter notebook, and a lot of developers use it for experiments when writing code. I can be useful to setup a local gitignore (only for you, not pushed to git) to have a playground where you can experiment, without needing to push files to git, or change the .gitignore. You can do this by adding a folder to your `.git/info/exclude` file, which works similar to the `.gitignore` file, e.g. if we add\n",
+    "```\n",
+    "notebooks/experimental/* \n",
+    "```\n",
+    "to `.git/info/exclude`, git wont sync the changes to the `experimental` folder to github"
+   ]
+  },
   {
    "cell_type": "markdown",
    "id": "518f1fa4-8d19-47f3-b6a4-725ec43b3300",
    "metadata": {},
    "source": [
     "## Working with Python Domain"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c2bc7af1",
+   "metadata": {},
+   "source": [
+    "PySyft enables a network of computers to connect to each other and do privacy preserving data analysis. The Nodes in the network that hold some data are called `Domains`. When we develop with PySyft, it is very common to start a domain as the first step. `PySyft` makes it very easy to develop against a domain in a notebook by providing an interface (`sy.orchestra`) that allows you to start a domain with a webserver in a notebook in the background, which is a lightweight version of a Domain that would be used in production. You can specify options such as what kind of database you are using, whether you want to use networking and how many processes you want to use. You can launch a Domain by simply executing:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3e54e427",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import syft as sy;"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6165cbad",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "node = sy.orchestra.launch(name=\"eager-test-domain-1\", port=\"auto\", reset=True);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "32cdb481",
+   "metadata": {},
+   "source": [
+    "Now, we are ready to start using the domain. The domain comes with standard login credentials for the admin (just for development)"
```

Comment:
Should we also add instructions that one can also start a pure Python based client (without uvicorn) by skipping `port` argument.

`node = sy.orchestra.launch(name="eager-test-domain-1", dev_mode=True, reset=True)`